fpga.chart.py

```diff
@@ -91,9 +91,12 @@ def __init__(self, configuration=None, name=None):
         for key in self.keys:
             self.default_data[key] = 0
 
-        self.os_thread = threading.Thread(target=self.get_fpga_os_status, args=())
-        self.os_thread.daemon = True
-        self.os_thread.start()
+        self.temp_thread = threading.Thread(target=self._get_fpga_temp, args=())
+        self.power_thread = threading.Thread(target=self._get_fpga_power, args=())
+        self.temp_thread.daemon = True
+        self.power_thread.daemon = True
+        self.temp_thread.start()
+        self.power_thread.start()
```

Comment:
Maybe put in a loop, e.g.

    for target in [self._get_fpga_temp, self._get_fpga_power]:
        start_thread(target)